.editorconfig

```diff
@@ -1,7 +1,16 @@
+root = true
+
 [*]
+charset = utf-8
+end_of_line = lf
```

Comment:
I use Windows and I want the line endings to remain CRLF when editing them.

Reply:
Fair enough. In other contexts that's a problematic default since it can interfere with e.g. testing mounted source in docker containers, or causing mismatched file hashes in supposedly-repeatable builds. But none of that applies here. And in any case the `.gitattributes` would need to be changed to really address those problems, too.